docs/setting-up-your-environment.md

```diff
@@ -97,13 +114,33 @@ To preview the website in your local browser, make sure you're in the
 `btcinformation.org` directory and run the following command:
 
     make preview
+    
+###### Mac differences:
+
+You might get an error that looks like this:
+```
```

Comment:
On my (non-Debian) linux system, trying to build also gives the error

```
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
                    Invalid US-ASCII character "\xE2" on line 1917
jekyll 3.0.1 | Error:  Invalid US-ASCII character "\xE2" on line 1917
```

So it seems this is not OSX specific, and therefore should be moved outside of the "Mac differences" part.

Reply:
I think that's a bug that should be dealt with separately (I'll look into it).  The makefile already should already be running locale independently:

```make
## Always build using the default locale so log messages can be grepped.
## This should not affect webpage output.
build:
        $S export LANG=C.UTF-8 ...
```

Reply:
Thought it was only my local environment, worth mentioning. Should I create a new issue for this?

Reply:
I think I have it fixed here; if you can test, that'd be great.  https://github.com/achow101/btcinformation.org/pull/19

Reply:
I removed the bug section from my original pull request. Please review.